api/routes/groups/groups.ts

```diff
@@ -92,6 +94,52 @@ router.get(
   },
 );
 
+router.patch(
+  '/joinEvent',
```

Comment:
I think it's better to use `/event/join` because later we could make `event/leave` and other routes dealing with the events of a group